app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/TableRowWidget/index.tsx

```diff
@@ -60,7 +60,7 @@ export default function({
           selectedColumns={selectedColumns}
           handleSingleSelection={handleSingleSelection}
           columnDetail={columnDetail}
-          handleDisableCheckbox={handleDisableCheckbox}
+          isCheckboxDisabled={handleDisableCheckbox}
```

Comment:
one of the following commits must have overwritten this?

Reply:
renamed handleDisableCheckbox to isCheckboxDisabled across files